### DISCUSSION TITLE
Bill co-pilot at the organization level

### DISCUSSION THREAD

**Discussion by AlexCLeduc** [community/community#29541] [Copilot Conversations]
As our co-pilots free-trials start to expire, I'm wondering if there are any plans to allow billing co-pilot at the organization level, rather than have developers pay for it personally? 

The simplest way to implement this would probably be to keep co-pilot access tied to users, rather than projects. I don't think most companies would care if some devs used co-pilot outside company projects. 
I suppose some companies may want to configure how many co-pilot licenses they buy, rather than pay for the whole organization 🤷

**Comment by brunoripa** [comment]
Looking for the same answer, and I also add: in case @AlexCLeduc gets a negative answer, the follow up question is: if a developer pays for Copilot on his own, will he be able to use Copilot on organization's repositories ?

**Comment by davidmondok** [comment]
Very interested in this as well. The current options with multiple bills, issued to personal accounts, several receipts, etc. are a mess for us as a company.

**Comment by jchen123** [comment]
I would imagine github must be treating this with some sense of priority -- copilots being available to individuals makes it totally unusable for orgs.  Lots of orgs allow people to use their personal github accounts -- the only way to pay for users as an org would be to pay with company cards individually (a shitshow when people leave the company) or messy reimbursements.
